data/spells/scripts/attack/inflict_wound.lua

```diff
@@ -3,16 +3,20 @@ combat:setParameter(COMBAT_PARAM_TYPE, COMBAT_PHYSICALDAMAGE)
 combat:setParameter(COMBAT_PARAM_EFFECT, CONST_ME_DRAWBLOOD)
 combat:setParameter(COMBAT_PARAM_DISTANCEEFFECT, CONST_ANI_WEAPONTYPE)
 
-function onTargetCreature(creature, target)
-	local skill = 0
-	for i = SKILL_CLUB, SKILL_AXE do
-		if skill < creature:getEffectiveSkillLevel(i) then
-			skill = creature:getEffectiveSkillLevel(i)
+function Creature:getHighestSkillLevel()
```

Comment:
Methods on the core metatables shouldn't be defined sparsely in the datapack. I suggest making this a local function instead.